app/services/doc_auth_router.rb

```diff
@@ -72,8 +72,7 @@ module DocAuthRouter
     IdentityDocAuth::Errors::GLARE_LOW_BOTH_SIDES => 'doc_auth.errors.glare.top_msg_plural',
   }.freeze
 
-  # Adds translations to responses from Acuant
-  class AcuantErrorTranslatorProxy
+  class DocAuthErrorTranslatorProxy
```

Comment:
Should `translate_form_response!` at least be stubbed out in the base class if we're expecting it to exist in `method_missing` ?

Reply:
Mmmm, good catch. Eventually everything will be moved into DocAuthErrorTranslatorProxy so that'll be fine. The impl will eventually just get moved in there. So the stub will be good for multiple things.